csrc/ops/composite.cpp

```diff
@@ -89,6 +89,40 @@ TensorView* matmul(TensorView* a, TensorView* b) {
   return matmul(a, b, true /* cast output to input dtype */);
 }
 
+TensorView* linear(TensorView* a, TensorView* b, TensorView* bias) {
+  NVF_CHECK(
+      a->nDims() == b->nDims(),
+      "The number of dimension of A and B do not match");
```

Comment:
This actually does not match [F.linear](https://pytorch.org/docs/stable/generated/torch.nn.functional.linear.html#torch.nn.functional.linear). `a` merely needs at least one dimension and `b` must have either 1 or 2 dimensions.